Enterprise-CMCS/macpro-mdct-seds
# Remove Excel export feature

### Description
SEDS has an admin-only page that displays a list of all users. That table has three export buttons, to download files of three different formats: Excel (.xlsx), CSV, and PDF.

This PR removes the Excel export option from that page. This eliminates our dependency on `exceljs`, an outdated package with security vulnerability warnings. While this vulnerability carried little-to-no risk for us, it will be nice to have the warnings gone.

![image](https://github.com/user-attachments/assets/f236ba8d-de28-4651-935d-5d4950a70b2c)
_look, no excel!_

### Related ticket(s)
CMDCT-3851
CMDCT-4056

---
### How to test
1. Log in to the app at https://dhsyk4pvmzqxb.cloudfront.net/
   * Use the adminuser@test.com account
2. Go to `View/Edit Users`
3. Note that there is no "Excel" export button above the table
4. (Optional): Note that the "CSV" button downloads a CSV with data from the table

### Notes
Removing the `exceljs` dependency reduced the size of app-api/node_modules by over half: from 67MB to 30MB

---
### Pre-review checklist


---
### Pre-merge checklist


#### Review

#### Security
_If either of the following are true, notify the team's ISSO (Information System Security Officer)._
